src/cbc_sdk/base.py

```diff
@@ -321,21 +321,42 @@ def __str__(self):
         lines.append("-" * 79)
         lines.append("")
 
-        for attr in sorted(self._info):
-            status = "   "
-            if attr in self._dirty_attributes:
-                if self._dirty_attributes[attr] is None:
-                    status = "(+)"
-                else:
-                    status = "(*)"
-            try:
-                val = str(self._info[attr])
-            except UnicodeDecodeError:
-                val = repr(self._info[attr])
-            if len(val) > 50:
-                val = val[:47] + u"..."
-            lines.append(u"{0:s} {1:>20s}: {2:s}".format(status, attr, val))
-
+        try:
```

Comment:
@llyon-cb @mrangelov-cb We can't add this code in both PRs otherwise we will get a conflict on merge. One of these PRs will need to manage this conflict after one of the PRs is merged. Please discuss who is going to fix the conflict and if any help is needed

Reply:
I reverted the change, as it doesn't break any tests.